modules/xfem/src/actions/XFEMAction.C

```diff
@@ -57,6 +57,11 @@ XFEMAction::validParams()
       "debug_output_level <= 3",
       "Controls the amount of debug output from XFEM.  0: None, 1: Summary, 2: Details on "
       "modifications to mesh, 3: Full dump of element fragment algorithm mesh");
+  params.addRangeCheckedParam<Real>("min_weight_multiplier",
+                                    1.e-3,
```

Comment:
why is it set to be 1.0e-3?

Reply:
If you already have found this optimal value, we should use it as the default value. Otherwise, users will have to waste their fine trying different values.